pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3314,3 +3361,18 @@ func (c *Cloud) addFilters(filters []*ec2.Filter) []*ec2.Filter {
 func (c *Cloud) getClusterName() string {
 	return c.filterTags[TagNameKubernetesCluster]
 }
+
+func setNodeDisk(
```

Comment:
maybe put some comments for this new function or use a more detailed function name?